Source/WebCore/rendering/RenderLayerBacking.h

```diff
@@ -419,6 +421,7 @@ class RenderLayerBacking final : public GraphicsLayerClient {
     RefPtr<GraphicsLayer> m_childContainmentLayer; // Only used if we have clipping on a stacking context with compositing children, or if the layer has a tile cache.
     RefPtr<GraphicsLayer> m_viewportAnchorLayer; // Only used on fixed/sticky elements.
     RefPtr<GraphicsLayer> m_maskLayer; // Only used if we have a mask and/or clip-path.
+    RefPtr<GraphicsLayer> m_transformSpacerLayer;
```

Comment:
"Spacer" doesn't really communicate this layer's role (it's not adding space). `m_transformFlatteningLayer` or just `m_flatteningLayer`?